addons/binding/org.openhab.binding.homie/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,26 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Homie Binding
+Bundle-SymbolicName: org.openhab.binding.homie;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .,lib/org.eclipse.paho.client.mqttv3-1.1.0.jar,lib/commons-lang3-3.5.jar
```

Comment:
1. As far as I know, there's only commons.lang build in, and not commons.lang3.
2. This would mean a complete reimplementation of the connection class and its architecture.